pkg/agents/team5/agent1/agent.go

```diff
@@ -49,6 +49,14 @@ func New(baseAgent *infra.Base) (infra.Agent, error) {
 	}, nil
 }
 
+func MaxHP(a *CustomAgent5) int {
+	return a.HealthInfo().MaxHP
+}
```

Comment:
Why not just call `a.HealthInfo().MaxHP` directly??

Reply:
i thought code would be easier to read, i can change if you like

Reply:
Ehhh we only use it in one place so far, I think we don't need the function - if we use a lot more and it gets annoying, then maybe yeah.